tests/unit/FindPark.spec.js

```diff
@@ -26,9 +26,18 @@ describe('FindPark', () => {
   it('should render headings on load', () => {
     const wrapper = mount(FindPark, { store, localVue });
     expect(wrapper.find('h2').text()).toBe("Let's Go Play!");
-    expect(wrapper.find('h3').text()).toBe('--Or--');
+    expect(wrapper.find('#search-switch').text()).toBe('--Or--');
   });
-  it('should render search button and input field on load', () => {
+
+  it('should disable Find Parks Near Me button if geolocation is inactive', () => {
+    const wrapper = mount(FindPark, { store, localVue });
+    expect(wrapper.find('.disabled').exists()).toBe(true);
+    // expect(wrapper.find('#location-message').text()).toBe(
+    //   'Turn on location services and reload the page to search for parks near you!'
+    // );
```

Comment:
The error you referenced in your PR seems strange.  Try putting line 36 as part of line 35 (it'll be long, but worth trying).  I'm wondering if it's picking up extra space somewhere outside of the string.

Reply:
The auto formatting on VSCode drops it down to that line. I also tried storing it to a variable and then calling the variable in the `toBe(msg)` like so and I got the same error.

Reply:
Turned off the auto formatting and now its passing!

Reply:
YES! That's good news for sure.